src/pages/ScheduleCall/ScheduleCallPage.tsx

```diff
@@ -76,21 +77,50 @@ function ScheduleCallPage() {
         if (!calendlySchedule?.data) {
             return {};
         }
-        const guides = Object.keys(calendlySchedule?.data);
+        const guides = Object.keys(calendlySchedule.data);
 
-        const allTimeSlots = guides?.reduce((allSlots, guideAccountID) => {
+        const uniqueSlotTimeSet = new Set<string>();
```

Comment:
Let's remove this logic, as it's filtered server side

Reply:
Both of them, as server handles it
1. Server won't sent duplicate timeslots
2. Server will prioritise assigned/primary guide.

We just need to sort the slots, so they are viewed in order